internal/stats/course.go

```diff
@@ -19,30 +19,14 @@ const (
 const (
```

Comment:
Is CourseMetricType needed anymore?
Its original point is that it is an enum and can be used as a type.
If the values are only going to be used in a general map, then there is not point in having a specific type for it.
If so, then they should just be constants instead of enums.